hefquin-base/src/test/java/se/liu/ida/hefquin/query/QueryPatternUtilsTest.java

```diff
@@ -0,0 +1,334 @@
+package se.liu.ida.hefquin.query;
```

Comment:
The test should be in the same package as the class that it tests.

Reply:
Fixed.